tests/test-sources/plugins/ui/repeat.nix

```diff
@@ -0,0 +1,17 @@
+{
+  empty = {
+    plugins.repeat.enable = true;
+  };
+
+  default = {
+    plugins.repeat = {
+      enable = true;
+    };
+  };
+
+  example = {
+    plugins.dracula = {
+      enable = true;
+    };
+  };
```

Comment:
```suggestion
```

Reply:
Damn, that a very shamefull copy/paste error on my part.